src/Test/modellike.jl

```diff
@@ -25,7 +25,7 @@ function default_status_test(model::MOI.ModelLike)
     @test MOI.get(model, MOI.DualStatus()) == MOI.NO_SOLUTION
 end
 
-function nametest(model::MOI.ModelLike)
+function nametest(model::MOI.ModelLike; delete::Bool=true)
```

Comment:
Shouldn't this take a `Test.Config`?

Reply:
I was hesitant because then we also need to take into account other fields such as `query`, etc... and that will be a lot of `if`-`else`. We could do this but we should first agree that's the direction we want to go since that might be a few changes